packages/react-components/react-dialog/src/contexts/dialogContext.ts

```diff
@@ -4,9 +4,18 @@ import { DialogSurfaceElement } from '../DialogSurface';
 import type { Context } from '@fluentui/react-context-selector';
 import type { DialogModalType, DialogOpenChangeData } from '../Dialog';
 import { useModalAttributes } from '@fluentui/react-tabster';
+import { MotionState } from '@fluentui/react-motion-preview';
 
 export type DialogContextValue = {
+  /**
+   * @deprecated Use motion instead
+   */
   open: boolean;
+  /**
+   * Motion is only optional because otherwise it would be a breaking change,
+   * but it should always be provided.
+   */
+  motion?: MotionState<DialogSurfaceElement>;
```

Comment:
It does not seem like an intuitive move for users to move from an `open` prop to a `motion` prop to control a component's visibility

Reply:
what would you suggest?